strapi/src/components/sections/banner-basic.json

```diff
@@ -0,0 +1,48 @@
+{
+  "collectionName": "components_sections_banner_basics",
+  "info": {
+    "displayName": "Banner Basic",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "content": {
+      "type": "text"
+    },
+    "textPosition": {
+      "type": "enumeration",
+      "enum": [
+        "left",
+        "right"
+      ],
+      "required": true
+    },
+    "media": {
+      "type": "media",
+      "multiple": false,
+      "required": true,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "header": {
```

Comment:
Please change to `title`.

Reply:
I also try to keep this schema in logical order, usually the same as it is displayed.
It's not crucial, but it makes life little bit easier :) (e.g. default fields order is correct)

So I would move title to top.